math/physics/inc/TVector3.h

```diff
@@ -34,6 +34,9 @@ class TVector3 : public TObject {
    TVector3(const Float_t *);
    // Constructors from an array
 
+   TVector3(Double_t phi, Double_t theta);
```

Comment:
What's Pt here?

Reply:
Sorry, what is pt? 😅

Reply:
Here, the two angles phi and theta allow you to make a transformation vector. If you have a point and two angles theta and phi, for example, you could use polar coordinates to get a transformation vector to calculate the trajectory. This function allows you to create a Cartesian Transformation Vector from two angles.